kw.sh

```diff
@@ -5,9 +5,53 @@ EASY_KERNEL_WORKFLOW=${EASY_KERNEL_WORKFLOW:-"kw"}
 src_script_path=${src_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src"}
 external_script_path=${external_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/external"}
 
+# Notification definitions
+declare -r ALERT_NONE=0
+declare -r ALERT_VISUAL=1
+declare -r ALERT_SOUND=2
+declare -r ALERT_BOTH=3
```

Comment:
It is not a good idea to have these variables here since we export all of them. I recommend you to move these variables to common.sh.